src/common/modules/data/Symbols.js

```diff
@@ -0,0 +1,285 @@
+"use strict";
+
+/**
+ * Symbols
+ *
+ * @public
+ * @const
+ * @type {Object.<string, string>}
+ */
+export const symbols = Object.freeze({
+	"--": "–",
+
+	"(e)": "€",
+	"...": "…",
+	"::": "∷",
+	":=": "≔",
+	"~=": "≅",
+	"+-": "±",
+	"-+": "∓",
+	"<<": "≪",
+	"<=": "≤",
+	"->": "→",
+	">=": "≥",
+	">>": "≫",
+	"<=>": "⇔",
+
+	// Fractions
+	"1/4": "¼",
+	"1/2": "½",
+	"3/4": "¾",
+	"1/7": "⅐",
+	"1/9": "⅑",
+	"1/10": "⅒",
+	"1/3": "⅓",
+	"2/3": "⅔",
+	"1/5": "⅕",
+	"2/5": "⅖",
+	"3/5": "⅗",
+	"4/5": "⅘",
+	"1/6": "⅙",
+	"5/6": "⅚",
+	"1/8": "⅛",
+	"3/8": "⅜",
+	"5/8": "⅝",
+	"7/8": "⅞",
+
+	// Fira Code (https://github.com/tonsky/FiraCode)
+	"<--": "⟵",
+	"<==": "⟸",
+	"==>": "⟹",
+	"-->": "⟶",
+	"<-->": "⟷",
+	"<--|": "⟻",
+	"<==|": "⟽",
+	"|-->": "⟼",
+	"|==>": "⟾",
+
+	"::=": "⩴",
+	"__": "＿",
+	"==": "⩵",
+	"!=": "≠",
+	"===": "⩶",
+	"<-<": "↢",
+	"<<-": "↞",
+	"<-": "←",
+	"<->": "↔",
+	"->>": "↠",
+	">->": "↣",
+	"<==>": "⟺",
+	"=>": "⇒",
+	"<=": "⇐",
+	">>-": "⤜",
+	">-": "⤚",
+	"<~>": "↭",
+	"-<": "⤙",
+	"-<<": "⤛",
+	"<-|": "↤",
+	"<=|": "⤆",
+	"/\\": "∧", // /\ 
+	"\\/": "∨", // \/
+	"|->": "↦",
+	"|=>": "⤇",
+	"<~~": "↜",
+	"<~": "↜",
+	"~~": "〰",
+	"~>": "↝",
+	"~~>": "↝",
+	"<<<": "⋘",
+	"<>": "◇",
+	">>>": "⋙",
+	"<|": "◁",
+	"|>": "▷",
+	"\\\\": "⑊", // \\
+	"||": "‖",
+	"..": "‥",
+	"---": "—",
+	"++": "⧺",
+	"+++": "⧻",
+	"-|": "⫞",
+	"_|_": "⫠",
+	"|-": "⊢",
+	"||-": "⊩",
+	"|=": "⊨",
+	"||=": "⊫"
+});
+
+/**
+ * Emojis
+ *
+ * @public
+ * @const
+ * @type {Object.<string, string>}
+ */
+export const emojis = Object.freeze({
+	"100": "💯",
+	"ZZZ": "💤",
+	"<!>": "⚠️",
+	"(-)": "⛔",
+	"(\\)": "🚫", // (\)
+	"??": "⁇",
+	"[]": "⬜",
+	"()": "⚪",
+	"!?": "⁉️",
+	"COOL": "🆒",
+	"[COOL]": "🆒",
+	"FREE": "🆓",
+	"[FREE]": "🆓",
+	"NEW": "🆕",
+	"[NEW]": "🆕",
+	"SOS": "🆘",
+	"[SOS]": "🆘",
+	"UP!": "🆙",
+	"[UP!]": "🆙",
+	"[BACK]": "🔙",
+	"[END]": "🔚",
+	"[ON!]": "🔛",
+	"[SOON]": "🔜",
+	"[TOP]": "🔝",
+	"[i]": "ℹ️",
+	"[OK]": "🆗",
+	"[VS]": "🆚",
+	"[x]": "❎",
+	"[*]": "*️⃣",
+	":+1:": "👍",
+	":-1:": "👎",
```

Comment:
They are here because they were emoticons before `:colon:` shortcodes existed, so users who have disabled `:colon:` shortcode autocorrections may still want them. I can remove them from here if you would like.

Reply:
Yes, please. Because I doubt that this is well-known (and no, you don't need to find a proof for it now). Maybe only if you write `+1` literally (without colons), but in this case that would belong into the "one-char" category again, so also not a thing we want to keep.
So yes, these emojis should be enabled with the `:colon:` shortcodes option…